OIIO/WriteOIIO.cpp

```diff
@@ -394,6 +402,7 @@ WriteOIIOPlugin::WriteOIIOPlugin(OfxImageEffectHandle handle,
     _bitDepth = fetchChoiceParam(kParamBitDepth);
     _quality = fetchIntParam(kParamOutputQuality);
     _dwaCompressionLevel = fetchDoubleParam(kParamOutputDWACompressionLevel);
+    _zipCompressionLevel = fetchIntParam(kParamOutputZIPCompressionLevel);
```

Comment:
- `zipquality` TIFF compression level
- OpenEXR 3.1.3+ `zip` and `zips` EXR compression level